backend/fms_core/models/project.py

```diff
@@ -0,0 +1,57 @@
+import reversion
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from .tracked_model import TrackedModel
+from django.contrib.auth.models import User
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._utils import add_error as _add_error
+from ._validators import name_validator, email_validator
+
+__all__ = ["Project"]
+
+
+@reversion.register()
+class Project(TrackedModel):
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH,
```

Comment:
We agreed on having a unique name, dont forget to add this

Reply:
My bad